python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -397,6 +397,13 @@ def main():
         sys.stderr.write('"multiprocessing" missing.\n')
         exit_code = 16
 
+    try:
+        import scandir
+        scandir.__file__
```

Comment:
Here, for both the .so and .pyc  there would be a `__file__`  ... and usually to have both pyc and .so the .so is creates with an underline.

```
>>> import ssl
>>> ssl.__file__
'/usr/lib/python2.7/ssl.pyc'
>> import _ssl
>>> _ssl.__file__
'/usr/lib/python2.7/lib-dynload/_ssl.x86_64-linux-gnu.so'
```